### PR TITLE
Updating mkdocs

### PR DESCRIPTION
**Pull request by nguyendankha**

### CODE REVIEW

**Comment by nguyendankha** [comment]
@trung-doan 
CC: @vonhunhu 
I'd finished the update mkdocs. please review my pull request (#8).

Let me know if anything make you feel concern.

Thank you.

File: docs/Reference/CheckBox.md [hunk 1]
```diff
@@ -825,6 +825,7 @@ export default class Plugin extends React.Component {
     }
 
     render() {
+        return
```
Comment (hokimcuc0202) [inline docs/Reference/CheckBox.md]: Syntax error:
return
         <div>
           <CheckBox items={this.state.items} value={this.state.value} onChange={(value) => {this.setState({value})}} />
           <button onClick={this.handleClick}>Click</button>
         </div>
Need change to:
return (
         <div>
           <CheckBox items={this.state.items} value={this.state.value} onChange={(value) => {this.setState({value})}} />
           <button onClick={this.handleClick}>Click</button>
         </div>
);

Comment (hokimcuc0202) [inline docs/Reference/CheckBox.md]: Syntax error:
return
         \<div> .....\<\/div>
Need change to:
return (
         \<div>  .....\<\/div>
);

File: docs/Reference/Dropdown.md [hunk 2]
```diff
@@ -306,7 +306,7 @@ var dropdown = new kintoneUIComponent.Dropdown({
 
 var body = document.getElementsByTagName("BODY")[0];
     body.appendChild(dropdown.render());
-
+var firstItem = dropdown.getItems()[0];
 dropdown.removeItem(0);
 console.log(firstItem);
```
Comment (hokimcuc0202) [inline docs/Reference/Dropdown.md]: Need change:
console.log(firstItem); => console.log(dropdown.getItems());

Comment (nguyendankha) [reply docs/Reference/Dropdown.md]: We no need to print all items. We just need to check the first item again when the old value is removed from list. 
This change is more suitable than all items printing.

Thank you

File: docs/Reference/CheckBox.md (hunk 1 above)
Comment (nguyendankha) [reply docs/Reference/CheckBox.md]: @hokimcuc0202 
That's my fault. I forget the '{}' for return statement. I had added it. 

Thank you a lot.

File: docs/Reference/Dropdown.md (hunk 2 above)
Comment (hokimcuc0202) [reply docs/Reference/Dropdown.md]: You're right!

**Pull request merged**